__tests__/Itinerary.test.js

```diff
@@ -0,0 +1,20 @@
+const Port = require('../src/Port.js');
+const Ship = require('../src/Ship.js');
+const Itinerary = require('../src/Itinerary.js');
+
+describe('Itinerary', () => {
+    it('can be instansiated', () => {
+        const liverpool = new Port('Liverpool')
+        const dublin = new Port('Dublin');
+        const itinerary = new Itinerary([liverpool, dublin]);
```

Comment:
You could make use of the beforeEach() lifecycle method and extract these to a more global scope, so beforeEach test you would create these instances and avoid repeating the same lines of code.

Also, I've noticed that in your next test you're using the jest.fn() as you should, so why not replicate the same logic for this one?